crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -361,6 +361,9 @@ where
             warn!(
                 target: "consensus::engine",
                 hook = %hook.name(),
+                head_block_hash = ?state.head_block_hash,
+                safe_block_hash = ?state.safe_block_hash,
+                finalized_block_hash = ?state.finalized_block_hash,
```

Comment:
```suggestion
                head_block_hash = %state.head_block_hash,
                safe_block_hash = %state.safe_block_hash,
                finalized_block_hash = %state.finalized_block_hash,
```

Reply:
then we'll get short version of hashes, no? And since we skipped this FCU, we want to know the hashes of the FCU we've skipped